### PR TITLE
Fix swagger validation errors

### PR DESCRIPTION
**Pull request by calbach**
Fixes master CI. I submitted before pulling/rebasing, so I didn't catch these newly introduced issues.

### CODE REVIEW

File: api/src/main/resources/workbench.yaml [hunk 1]
```diff
@@ -754,7 +754,6 @@ paths:
           description: specifies which cdr version
         - in: body
           name: request
-          type: integer
           required: true
           description: size of the participant cohort statuses sample
           schema:
```
Comment (danqrodney) [inline api/src/main/resources/workbench.yaml]: The description here is wrong. @freemabd1 may want to fix in a follow-up.

Comment (freemabd1) [reply api/src/main/resources/workbench.yaml]: will do

Comment (calbach) [reply api/src/main/resources/workbench.yaml]: Eh, may as well fix while I'm touching this line. Done.

Comment (freemabd1) [reply api/src/main/resources/workbench.yaml]: thanks

**Pull request merged**